apps/bucket/cloudformation.yml

```
@@ -162,6 +159,6 @@ Resources:
         - Action: s3:PutObject
           Effect: Allow
           Principal:
-            AWS: !Sub arn:aws:iam::${AwsApplicationAccountId}:role/${FetcherRoleName}
+            AWS: !Sub arn:aws:iam::${AwsApplicationAccountId}:root
```

Per https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html#principal-accounts , you can alternatively just give the account id here

```suggestion
            AWS: !Ref AwsApplicationAccountId
```